examples/MvcWeb/TheraLang/ProjectStatusId.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace MvcWeb.TheraLang
+{
+    public enum ProjectStatusId
+    {
+        New = 0,
+        Approved = 1,
+        Rejected = 2
```

Comment:
```suggestion
        Rejected = 20
```
This is a good practice to make the difference in 10 - because we can have other statuses in between later